src/software/simulated_tests/ball_speed_threshold.py

```diff
@@ -17,6 +18,8 @@ def __init__(self, speed_threshold):
         :param speed_threshold: The speed threshold in m/s
         """
         self.speed_threshold = speed_threshold
+        self.MILLIMETERS_PER_METER = 1000
```

Comment:
you have access to this from `from software.py_constants import *` and then you can just use all the constants in py_constants.cpp